cmd/deploy/deploy.go

```diff
@@ -0,0 +1,58 @@
+package deploy
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/stormkit-io/stormkit-cli/api"
+	"github.com/stormkit-io/stormkit-cli/cmd"
+	"github.com/stormkit-io/stormkit-cli/model"
+	"github.com/stormkit-io/stormkit-cli/stormkit"
+)
+
+// deployCmd represents the deploy command
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "A brief description of your command",
```

Comment:
**suggestion** 

`Short: "Initiate a deployment process"`

`Long: "Initiate a deployment process by providing the environment and branch names. The specified environment will be used to read the configuration to use while building the given branch."`

Reply:
Done [39a9d71](https://github.com/stormkit-io/stormkit-cli/pull/9/commits/39a9d71c88c766cfc5000536ad8e64d98826c3cc)